.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug Report
+about: Report a bug or unexpected behavior
+title: ""
+labels: bug
+assignees: ''
```

Comment:
```suggestion
assignees: 'superlopuh'
```

Reply:
You want to be assigned to every bug?

Reply:
yep, I'm already subscribed to everything that happens, but I can always reassign to someone else

Reply:
This contradicts unassigned bugs and also constitutes a bit of a single point of failure.
Moreover, if you're subscribed to everything, you can do the "caretaking" when dealing with this notification, right?

Reply:
Yep fair enough